bpf/kprobe_pwru.c

```diff
@@ -366,7 +366,7 @@ handle_everything(struct sk_buff *skb, struct pt_regs *ctx, bool has_get_func_ip
 #endif /* HAS_KPROBE_MULTI */
 
 #define PWRU_ADD_KPROBE(X)                                                     \
-  SEC(PWRU_KPROBE_TYPE "skb-" #X)                                              \
+  SEC(PWRU_KPROBE_TYPE "/skb-" #X)                                             \
```

Comment:
Could you squash this commit into the [use macros to add krpobes](https://github.com/cilium/pwru/pull/136/commits/9dcd73fd11b179e576073ba9fe59a7261a4d65a9)?